index.html

```diff
@@ -289,6 +289,33 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+<section class="contact-me">
+  <div>
+  <p class="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+  </div>
+  <div>
+  <form action="https://formspree.io" method="post">
```

Comment:
- Kindly fix the form action as requested  by the previous reviewer. Currently your form is not being submitted, it returns an error, shown by the screenshot the previous reviewer has shown you. 
  
Below is a simple guide that can assist you
<img width="616" alt="Screenshot 2022-09-16 at 11 22 52" src="https://user-images.githubusercontent.com/48546992/190593038-675bd30a-129a-4188-884c-21557c0e01b3.png">

Notice that your action link is missing the {form id}